ote/src/cljs/ote/app/controller/route/route-wizard.cljs

```diff
@@ -188,13 +222,15 @@
                            ::transit/port-type
                            ::transit/port-type-name
                            ::transit/type)
-          new-stop-sequence (if (= (::transit/code (last stop-sequence)) (get  properties "code"))
+          new-stop-sequence (if is-stop-added?
                               stop-sequence
-                              (conj stop-sequence new-stop))]
+                              (conj stop-sequence new-stop))
+          new-trip-sequence (if is-stop-added?
+                              (get-in app [:route ::transit/trips])
+                              (calculate-trip-sequence new-stop-idx new-stop (get-in app [:route ::transit/trips])))]
```

Comment:
I would remove `calculate` and `calc` prefix verbs from function names, most of what we do is calculation.